Duplicati/Server/WebServer/AuthenticationHandler.cs

```diff
@@ -179,14 +179,19 @@ public override bool Process(HttpServer.IHttpRequest request, HttpServer.IHttpRe
                         return true;
                     }
 
+                    var password = Program.DataConnection.ApplicationSettings.WebserverPassword;
+
+                    if (request.Headers["User-Agent"] != null && request.Headers["User-Agent"].StartsWith("Duplicati TrayIcon Monitor"))

```

Comment:
We need a better way to do this. Relying on the `User-Agent` header seems dodgy.

We can either have an alternative `LOGIN_SCRIPT_URI`, or use a custom header like `X-TrayIcon-Login`.